libusb-game-controller/usb.h

```diff
@@ -0,0 +1,185 @@
+#ifndef _USB2_H_
+#define _USB2_H_
```

Comment:
```c
#ifndef UGC_USB_H
#define UGC_USB_H
```
Or maybe should we just switch everything to `#pragma once`? 🤔

Reply:
I'm for #ifndef, but don't have a strong opinion.